src/test/java/seedu/address/model/session/InvalidStartEndTimeTest.java

```diff
@@ -12,10 +11,9 @@
 
     @Test
     public void constructor_invalidTime_throwsIllegalArgumentException() {
-        LocalTime startTime = LocalTime.of(14, 00, 00);
-        LocalTime endTime = startTime.minus(1, ChronoUnit.HOURS);
-        LocalDate startDate = LocalDate.of(2020, 01, 01);
-        assertThrows(IllegalArgumentException.class, () -> new Session(startTime, endTime, startDate,
+        LocalDateTime startTime = LocalDateTime.of(2020, 01, 01, 14, 00, 00);
+        LocalDateTime endTime = startTime.minus(1, ChronoUnit.HOURS);
+        assertThrows(IllegalArgumentException.class, () -> new Session(startTime, endTime,
                 Session.SessionType.SESSION_TYPE_OTHER, ""));
```

Comment:
We could keep the old test case as well, since we are working with DateTime now. What do you think?

Reply:
I think unit-tests should behave the exact same way each time it is triggered, and not be dependent on the current time.